libs/blocks/global-navigation/global-navigation.js

```diff
@@ -479,6 +482,35 @@ class Gnav {
     return decoratedElem;
   };
 
+  decorateAside = async () => {
```

Comment:
Should we have a dynamic import for this, since it's not always used by everyone?

Or at least leave a comment+ticket so we can refactor this if we decide on a different path later on?

Reply:
We'd still need some base logic here, since we need to check the metadata property and decide whether we need to import the Aside module. So essentially we'd be making an additional call for the code on L490-L509. I'm not sure it's completely worth it, wdyt?

Reply:
It's a though one, if it grows certainly own module - for now we can keep it